frontend/src/components/common/form/TimePicker.tsx

```diff
@@ -34,6 +34,9 @@ const chakraStyles = (
       outline: "none",
     },
     border: "none",
+    _disabled: {
+      opacity: 1,
```

Comment:
what's this for?

Reply:
Due to an implementation detail the `opacity`-based styling for disabled elements was getting applied twice, meaning that this component looked different than other disabled components. Technically this fix belongs in the previous pr but that's already merged so I thought this was fine.